schemas/mw-page-delete.yaml

```diff
@@ -0,0 +1,53 @@
+title: MediaWiki Page Delete
+description: Represents a MW Page Delete event
+type: object
+properties:
+  # global event fields
+  event:
```

Comment:
Another option might be `head`, after the html element. I was also looking at http://stateless.co/hal_specification.html to see if we can steal any ideas there (esp. `_links`, but our head / meta / envelope info is a mix of references and data. We could shoehorn that data into URLs, but it doesn't feel super natural.